examples/6-simple-reduce-pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: even-odd-sum
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+    - name: even-or-odd
+      udf:
+        container:
+          # Tell the input number is even or odd, see https://github.com/numaproj/numaflow-go/tree/main/pkg/function/examples/evenodd
+          image: quay.io/numaio/go-even-odd-example
+    - name: compute-sum
+      udf:
+        container:
+          # compute the sum
+          image: quay.io/yhl25/reduce-image:master
```

Comment:
nit: should we use the `/numaio/` path and add a reference to the source of of the image? (same as the go-even-odd-example above.)

Reply:
+1, I have sent an invite to @yhl25 to join quay.io/numaio.

Reply:
yes, will do that. I didn't have access o **numaio** so I was using my personal repo.